wasm/README.md

```
@@ -0,0 +1,223 @@
+```
+
+### Running Tests
+
+```bash
+# From repository root
+pytest tests/test_wasm_build.py -v
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)